addon/-legacy-private/system/relationships/state/relationship.js

```diff
@@ -554,7 +554,8 @@ export default class Relationship {
 
     this.setHasFailedLoadAttempt(false);
     this.setShouldForceReload(true);
-    this.getData(true);
+    options.reload = true;
+    this.getData(options);
```

Comment:
let's make this `this.getData(options, true)`. We don't want to leak the ability to add `reload` as an option to calls to `load`, and the reason for this flag is to enforce that only this specific call to `getData` is considered a `reload`.